src/snowflake/cli/api/exceptions.py

```diff
@@ -194,24 +193,18 @@ def __init__(self, options: list[str]):
 class NoWarehouseSelectedInSessionError(ClickException):
     def __init__(self, msg: str):
         super().__init__(
-            dedent(
-                f"""\
-                        Received error message '{msg}' while executing SQL statement.
-                        Please provide a warehouse for the active session role in your project definition file, config.toml file, or via command line.
-                        """
-            )
+            "Received the following error message while executing SQL statement:\n"
```

Comment:
why?

Reply:
The error message can contain line breaks, so it breaks dedent when there is a new line without any indents